packages/java/src/browser/java-client-contribution.ts

```diff
@@ -44,9 +48,23 @@ export class JavaClientContribution extends BaseLanguageClientContribution {
 
     protected onReady(languageClient: ILanguageClient): void {
         languageClient.onNotification(ActionableNotification.type, this.showActionableMessage.bind(this));
+        languageClient.onNotification(StatusNotification.type, this.showStatusMessage.bind(this));
         super.onReady(languageClient);
     }
 
+    protected showStatusMessage(message: StatusReport) {
+        window.clearTimeout(this.statusBarTimeout);
+        const statusEntry = {
+            alignment: StatusBarAlignment.LEFT,
+            priority: 1,
+            text: message.message
```

Comment:
suggestion: if we prepend it with `$(refresh~spin)`, the progress is getting more visible.